app/controllers/message_objects_controller.rb

```diff
@@ -19,17 +19,17 @@ def update
 
   def show
     authorize @message_object
-    send_data @message_object.content, filename: MessageObjectHelper.displayable_name(@message_object), type: @message_object.mimetype, disposition: :inline
+    send_data @message_object.content, filename: MessageObjectHelper.displayable_name(@message_object), type: @message_object.mimetype || 'application/octet-stream', disposition: :inline
```

Comment:
nechceme radsej nastavit ten mimetype na octet stream vsade kde nevieme lepsie?

Reply:
Hej, to asi bude lepsie